rest-service/manager_rest/storage/resource_models.py

```diff
@@ -910,6 +905,9 @@ def blueprint(cls):
             'Blueprint', lazy='joined',
```

Comment:
Why did you leave quotes here?

Reply:
removed now!

It doesn't really matter, anyway, sqlalchemy fetches them by string as well (it does it specifically so that it's possible to have two models reference each other, it'd be a NameError in one of them otherwise)
(but in this case, BlueprintLabel is defined after Blueprint, so it doesn't need to go by string)